src/components/app/pageBills/index.tsx

```diff
@@ -2,17 +2,21 @@ import { partition, sortBy } from 'lodash-es'
 
 import { CryptoSupportHighlight } from '@/components/app/cryptoSupportHighlight'
 import { DTSIBillCard } from '@/components/app/dtsiBillCard'
+import { PageSubTitle } from '@/components/ui/pageSubTitle'
 import { PageTitle } from '@/components/ui/pageTitleText'
 import { SWCBillCardInfo } from '@/data/bills/types'
 import { SupportedCountryCodes } from '@/utils/shared/supportedCountries'
+import { cn } from '@/utils/web/cn'
 
 interface PageBillsProps {
   bills: SWCBillCardInfo[]
   countryCode: SupportedCountryCodes
+  description?: string
+  title?: string
 }
 
-export function PageBills(props: PageBillsProps) {
-  const { bills, countryCode } = props
+export function PageBills({ bills, countryCode, description, title }: PageBillsProps) {
+  const hasWrapper = title && description
```

Comment:
the prefix "has" suggests a boolean value, but this `hasWrapper` can be a string
<img width="149" height="50" alt="Image" src="https://github.com/user-attachments/assets/df04aeb2-533c-4e63-86b5-4f34fd438371" />

can we wrap this with a Boolean constructor or use `!!`?

```suggestion
  const hasWrapper = Boolean(title && description)
```

```suggestion
  const hasWrapper = !!title && !!description
```